deploy/kubernetes/examples/demo-storageclass-using-secret-from-configuration.yaml

```diff
@@ -0,0 +1,25 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: demo-storageclass
+provisioner: block.csi.ibm.com
+volumeBindingMode: WaitForFirstConsumer  # Optional. Required for Topology Awareness support.
+parameters:
+  # a pool value MUST be provided for all systems.
+  # pool, SpaceEfficiency and volume_name_prefix can be provided in by_system_id parameter for each system or by parameter name as default.
+
+  by_system_id: '{"demo-system-id1":{"pool":"demo-pool","SpaceEfficiency":"thin","volume_name_prefix":"demoPVC"},
+                  "demo-system-id2":{"pool":"demo-pool2","volume_name_prefix":"demoPVC2"}}'  # Optional.
+  pool: demo-pool                # Optional.
+  SpaceEfficiency: deduplicated  # Optional.
+
+  csi.storage.k8s.io/provisioner-secret-name: demo-secret-with-config
+  csi.storage.k8s.io/provisioner-secret-namespace: default
+  csi.storage.k8s.io/controller-publish-secret-name: demo-secret-with-config
+  csi.storage.k8s.io/controller-publish-secret-namespace: default
+  csi.storage.k8s.io/controller-expand-secret-name: demo-secret-with-config
+  csi.storage.k8s.io/controller-expand-secret-namespace: default
+
+  csi.storage.k8s.io/fstype: xfs   # Optional. Values ext4\xfs. The default is ext4.

```

Comment:
any reason to have 3 spaces and not 2, before the number sign?

Reply:
it's aligned with the comment below

Reply:
any reason to have 6 space and not 5, before the number sign, in the comment below?

Reply:
aligned